compose/ui/ui-graphics/build.gradle

```diff
@@ -94,7 +94,7 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
             skikoMain {
                 dependsOn(commonMain)
                 dependencies {
-                    api(libs.skikoCommon)
+                    implementation(libs.skikoCommon)
```

Comment:
Could you also add a section `API Change` to the message? The plan is to make a link to this PR in the CHANGELOG.

Reply:
https://github.com/JetBrains/compose-multiplatform/pull/3370